CONTRIBUTING.md

```diff
@@ -44,6 +44,13 @@ dependencies.
    Hence, to run these tests, you will need Python 2.7 and Python 3.x
    (preferably 3.6+) interpreters installed on your system.
 
+Required Python packages are listed in [requirements.txt](requirements.txt) file
```

Comment:
Suggested rewording for clarity:

Required Python packages are listed in **the** [requirements.txt](requirements.txt) file in this repository. **They can be installed** ~This can be done~ with pip with the following command:.